docs/open-source-project/release-notes/tizen-5-0-m1.md

```diff
@@ -250,11 +248,11 @@ Release date: May. 31, 2018
 ### New and changed features
 
 - EFL
-  - Version 1.16 is upgraded to 1.20
-  - Edbus and Eldbus is removed for security reasons.
+  - Version 1.16 is upgraded to 1.20.
+  - EDbus and Eldbus are removed for security reasons.
   - Elementary repository is integrated into EFL repository.
   - Added SVG morphing animation.
-  - Ecore_wayland is replaced with ecore_wayland2.
+  - ecore_wayland is replaced with ecore_wayland2.
   - Added GBS incremental build support.
 - Fontconfig
   - Version 2.12.1 is upgraded to 2.13.

```

Comment:
In line 271, use either applied or updated..

Applied reference TV IME GUI option in UI.
(or)
Updated the reference TV IME GUI option in UI.

Reply:
changed to make correct meaning.

Applied the enhanced reference TV IME GUI option in UI.